flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -216,6 +217,10 @@ class AndroidNotificationDetails {
   /// Refer to Android notification API documentation at https://developer.android.com/reference/androidx/core/app/NotificationCompat.html#constants_2 for the available categories
   final String category;
 
+  /// Specifies whether the notification should launch a full-screen intent as soon as it triggers.
+  /// Note: The system UI may choose to display a heads-up notification, instead of launching your full-screen intent, while the user is using the device.
```

Comment:
A blank line should separate the two sentences like the rest of the API docs to follow [these guidelines](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph)